plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -93,8 +96,12 @@ servers = [
 ## A list of queries to include. If not specified, all the above listed queries are used.
 # include_query = []
 
-## A list of queries to explicitly ignore.
-# exclude_query = []
+## A list of queries to explicitly ignore. Query Store queries are excluded by default.
+# exclude_query = [AzureSQLDBQueryStoreRuntimeStatistics, AzureSQLDBQueryStoreWaitStatistics]
+
+## Toggling this to true enables telegraf to start collecting data from query store.
+## This setting is optional and is disabled by default as it the queries are expensive and collection interval should be >=15m
+# query_store_collection = false
```

Comment:
is this something that can be covered by the exclude queries without adding a new boolean flag?

Reply:
@ssoroka, justification from @denzilribeiro, why do we need this flag, available at the following link: [https://github.com/influxdata/telegraf/pull/8465#issuecomment-823711117](https://github.com/influxdata/telegraf/pull/8465#issuecomment-823711117)

Reply:
I think the question is more can we have something like
```toml
enabled_features = ["query_store_collection",...]
```
to not add a list of booleans as new features are added.

Reply:
@srebhan : On second thought to what @ssoroka called out, we can just use exclude queries without adding a new boolean flag. This means that for old config + new telegraf bits, QDS data would start to collect. Is that an acceptable scenario and can we add extra documentation for users to take latest config to avoid such scenarios? I agree that adding too many optional parameters is not scalable and would it be possible to just use the existing exclude property instead of another property enabled_features as well. let us know your thoughts

Reply:
@masree I think you should better use `include_query` instead of `exclude_query` to guarantee an _explicit_ opt-in. Otherwise, people that have an existing `exclude_query` list will *automagically* get a new query which is surely not intended (given that they use the option in the first place). But yeah reusing that mechanism was exactly the point of @ssoroka...

Reply:
Yea, I think with the existing include/exclude parameter, we should be able to achieve this behavior rather than introducing another property to control the same